### ISSUE TITLE
Add background sync

### ISSUE THREAD

**Issue by trymeouteh** [etesync/ios#23]
I am using etesync on Android and I got my parents using etesync on iOS. I know the Android app has been around longer than the iOS app.

The Android app has a settings to allow you to choose how often your phone syncs with the etesync servers. I set mine to 15 minutes. There is no option on iOS that I can see and my parents can wait days and will not get the new events I added in the calendar unless they manually open etesync and press the sync button. Same goes for events my parents add, they need to create the events, open the app and press sync in order for me to get the events on my device.

My parents iphones still have iCloud enable so etesync is using the fake dav setup.

**Comment by jtrees** [comment]
I'm also waiting for this feature. Is there an ETA on when it will land?

**Comment by devurandom** [comment]
It is difficult to convince people to use "alternative software" if they perceive it as "it does not work at all", because a feature that is expected from a calendar / todolist synchronisation app is not available.

**Comment by tasn** [comment]
@devurandom, I agree. Though the problem is that Apple is user and developer hostile and doesn't let developers do almost anything, and is limited with what users can do to give apps extra capabilities.
We, like every other iOS developer, need to be creative with solutions that will both work and pass app review. We had a few good ideas there, though unfortunately the Apple imposed limitations were more strict than anticipated. We have a few more ideas, especially with new API added in iOS 13 that may work, but it requires a bit more work before we can actually start using it. :|

**Comment by avajaimieturner** [comment]
+1

To be perfectly frank, though I love what the service is trying to do, given it’s limited scoped (I don’t really need contact syncing, and there don’t appear to be any iOS apps that work with reminder syncing) $2/month would be a little pricey even if everything worked flawlessly. But it’s definitely too expensive for me when calendars have to be synced manually.

**Comment by ryan77627** [comment]
Hey, I just found this software. Any updates on iOS background syncing? I love it on everything else, if it had background sync for iOS it would be pretty much perfect for my use case. :)

**Comment by abramovkg** [comment]
Wondering if there have been any updates in iOS that would make implementing this feature easier to do? I am already a user hoping to use with my family and this is one thing that is holding me back.
